lib/modules/dosomething/dosomething_signup/dosomething_signup.signup_data_form.inc

```diff
@@ -55,6 +55,7 @@ function dosomething_signup_node_signup_data_form($form, &$form_state, $node) {
     '#weight' => 60,
     '#collapsible' => TRUE,
     '#collapsed' => FALSE,
+    '#description' => t('Note: you must either chose Competition Signup or Enable Custom Share Modal, cannot have both.'),
     // Config fieldset should only be visible if the status field is checked.
```

Comment:
Note: chose not to make Competition Signup and Enable Custom Sharing Modal as radio buttons and keep as checkboxes in case we want functionality to have them both to be enabled in the future.